.evergreen/install-node.sh

```diff
@@ -4,6 +4,11 @@ export BASEDIR="$PWD/.evergreen"
 
 if [ "$OS" == "Windows_NT" ]; then
   powershell "$(cygpath -w "$BASEDIR")"/InstallNode.ps1
+
+  # Explicitly grab a fresh portable Git for Windows build
+  curl -L https://github.com/git-for-windows/git/releases/download/v2.32.0.windows.2/MinGit-2.32.0.2-busybox-64-bit.zip -o "$BASEDIR/mingit-2.32.0.zip"
+  mkdir "$BASEDIR/mingit"
+  unzip "$BASEDIR/mingit-2.32.0.zip" -d "$BASEDIR/mingit"
```

Comment:
By the way, this means that we can migrate back from windows-64-vs2019-small-git-latest to windows-64-vs2019-small (i.e. revert 1503a30c294756bcd7009f8817185121204265f9), I think :)

Reply:
Right I'll change that immediately, too